armi/runLog.py

```diff
@@ -503,9 +515,41 @@ class RunLogger(logging.Logger):
         :id: I_ARMI_LOG
         :implements: R_ARMI_LOG
 
+        This logger makes it easy for users to add log statements to
+        and ARMI application, and ARMI will control the flow of
+        those log statements. In particular, ARMI overrides the normal
+        Python logging tooling, to allow developers to pipe their log
+        statements to both screen and file. This works for stdout and
+        stderr.
```

Comment:
I think these two sentences should be moved to the LOG_IO impl, since it's directly relevant to that.